cfg/pre-commit-config.yaml

```diff
@@ -109,6 +109,16 @@ repos:
     name: 'mypy'
     types:
     - 'python'
+  -
+    entry: |
+      pnpm run lint
+    files: 'src/typescript'
+    id: 'pnpm'
+    language: 'node'
+    name: 'lint typescript'
+    pass_filenames: false
+    types:
+    - 'directory'
```

Comment:
does this run on all files? I can't find `language: node` in `pre-commit` config on the site

Reply:
This only run once. Since `pnpm run lint` already checks all the files.